guides/python-testing.md

```diff
@@ -100,25 +100,18 @@ push events (e.g., commits to GitHub) or to run in a preset cadence
 
 For GitHub, here are some services available:
 
+* GitHub Actions: This is a paid service billed centrally to STScI.
+  Repositories on ``spacetelescope`` GitHub organization should use it,
+  if possible.
 * Jenkins CI: This is an in-house CI solution with virtually no limitation
   (but do not push your luck). It supports storing big data and artifacts in
   Artifactory. Its internal-only instance also supports direct access to
```

Comment:
The system does not allow for direct access to central storage.

Reply:
I don't understand your suggested change here. Can you please clarify?

Reply:
The server doesn't allow you to access central storage at all.

Reply:
I think the internal-only instance is referring to `JenkinsfileRT`.

Reply:
How about:

If STScI staff need to execute testing at regular intervals, or require direct access to central storage, please refer to the following documentation, [Users Guide: Running Regression Tests](https://innerspace.stsci.edu/display/SSR/Users+Guide%3A+Running+Regression+Tests) for more information.

Reply:
Your text is applied. What about now?